server/tabby_server/vision/yolo_test.py

```diff
@@ -0,0 +1,81 @@
+from flask import Blueprint, request
+from http import HTTPStatus
+from ultralytics import YOLO
+
+
+# Blueprint for this file. Adds the 'test' prefix in __main__.py.
+# To call functions in this file use "/test/<function_route>"
+yolo_test = Blueprint("yolo_test", __name__)
+
+
+"""
+This file is a testbed implementation of You Only Look Once (YOLO) image
+recognition. Credits at the bottom.
+"""
+
+
+# Instantiates the model.
+# Runs it. Outputs files to vision/example_yolo.
+
+
+def ultralytics_shelf_detection(file_path, save_output):
+    # Load pretrained model.
+    model = YOLO(file_path + "shelf_yolo.pt")
+
+    # Use model on given image url and return it.
+    # Outputs the resulting image to vision/example_yolo.
+    model(
+        source=(file_path + "example_shelves"),
+        conf=0.45,
+        save=save_output,
+        project=(file_path + "example_yolo"),
+    )
+
+
+# Callable.
+# Calls the model for each example file in vision/example_shelves.
+# Doesn't return anything of value to HTTP.
+# Use parameter index to indicate if you want to save the output or not.
+
+
+@yolo_test.route("/shelf_read", methods=["GET"])
+def predict_examples():
+    # Get parameter index.
+    # 1 means you want to save (default). 0 means you don't.
```

Comment:
This looks like it should be replaced with a bool data type instead of integer. Is there a reason we have to use int instead of bool for true and false?